src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { ALERT_SIZE, ALERT_TYPE, ALERT_ICON } from "./enums";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: "alert",
+          intro:
+            "Use the alert (or notice) component to keep the user updated about important information.",
+        }),
+      },
+    },
+  },
+  args: {
+    title: "Opgelet!",
```

Comment:
Ik zou de default gewoon title `Lorem ipsum` en content een iets langere lorem ipsum geven, met gewoon het info icon. Dat lijkt me het meest standaard. Nu is de default eigenlijk de Error zonder de juiste type, dat is wat verwarrend.

Reply:
Vergelijkbaar met https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/native-elements-vl-introduction--default

Reply:
Of beter nog, aangedaan de info-type default is, zou ik de Default story misschien gewoon weglaten en starten met de Info story.

Reply:
Ik merk dat je er toch een verschil is met geen type meegeven, of de info type meegeven. Is info type dan wel default zoals je nu in de documentatie hebt gezet? Als dat niet het geval is, zou ik de waarde van de type in de default story leeglaten onload, dan kan de gebruiker nog steeds togglen. zoals bv: https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/native-elements-vl-icon--default data-vl-size

Reply:
Het is de default. Indien je deze niet meegeeft krijg je een andere styling die niet echt klopt lijkt me... Dus info moet hier standaard aan staan toch.

Reply:
Maar aangezien die niet standaard 'aan' staat is het niet de default. Als we willen dat die default is, moeten we dat implementeren in de component, voorlopig is het anders documenteren fout, anders kunnen teams er van uit gaan dat als ze de type prop niet meegeven dat de de info alert krijgen